server/src/main/java/org/apache/iotdb/db/auth/authorizer/OpenIdAuthorizer.java

```diff
@@ -189,7 +188,7 @@ private Claims validateToken(String token) {
         .setAllowedClockSkewSeconds(Long.MAX_VALUE / 1000)
         // .setSigningKey(DatatypeConverter.parseBase64Binary(secret))
         .setSigningKey(providerKey)
-        .parseClaimsJws(token)
+        .parseClaimsJws(token.replaceAll("access_token:", ""))
```

Comment:
What does this line do?

Reply:
I'm sorry, I take the return value( eg:{"access_token":xxx, "token_type": "Bearer","not-before-policy": 0, "session_state": "xxx","scope": "profile email","expires_in": 300,"refresh_expires_in": 1800,"refresh_token": ""} )as the token.

Reply:
I'm sorry, I take the return value( eg:{"access_token":xxx, "token_type": "Bearer","not-before-policy": 0, "session_state": "xxx","scope": "profile email","expires_in": 300,"refresh_expires_in": 1800,"refresh_token": ""} )as the token.